typos.toml

```diff
@@ -0,0 +1,41 @@
+[files]
+extend-exclude = [
+    # ./compiler excludes
+    "compiler/rustc_codegen_gcc",
+    "compiler/rustc_codegen_cranelift",
+    "compiler/rustc_baked_icu_data",
+]
+
+[default.extend-words]
+thir = "thir"
+padd = "padd"
+rplace = "rplace"
+arange = "arange"
+rela = "rela"
+unstalled = "unstalled"
+taits = "taits"
+Datas = "Datas"
+splitted = "splitted"
+leafs = "leafs"
+Lits = "Lits"
+makro = "makro"
+optin = "optin"
+parm = "parm"
+EXCED = "EXCED"
+Shortern = "Shortern"
+unparseable = "unparseable"
+replacable = "replacable"
```

Comment:
Is that not a typo of "replaceable"?

Reply:
Internet says that correct variant is `replaceable` (even my spellchecker in browser), but `replacable` still used. Will fix then.